client/index.html

```diff
@@ -205,6 +205,13 @@ <h1 class="title">
 							<div class="col-sm-12">
 								<h1 class="title">Settings</h1>
 							</div>
+							<% if (displayConnection === true) { %>
```

Comment:
Because i struggled with html and css side, I hide it by default. Some help with design would be nice.